src/backend/distributed/utils/metadata_cache.c

```diff
@@ -339,10 +365,12 @@ CitusHasBeenLoaded(void)
 Oid
 DistShardRelationId(void)
 {
-	static Oid cachedOid = InvalidOid;
+	Oid cachedOid = metadataCacheState.distShardRelationId;
```

Comment:
Onder pulled this up in code review for my compressed_fdw.

What's the argument for this being more readable than just using the struct directly?